describe_configs_request.go

```diff
@@ -0,0 +1,91 @@
+package sarama
+
+type Resource struct {
```

Comment:
CI is failing because there is already a type named Resource being used for the ACL stuff